src/ResourceManager/Insights/Commands.Insights/Alerts/RemoveAzureRmAlertRuleCommand.cs

```diff
@@ -15,13 +15,14 @@
 using Microsoft.Azure.Management.Insights;
 using System.Collections.Generic;
 using System.Management.Automation;
+using Microsoft.Azure.Management.Insights.Models;
 
 namespace Microsoft.Azure.Commands.Insights.Alerts
 {
     /// <summary>
     /// Remove an Alert rule
     /// </summary>
-    [Cmdlet(VerbsCommon.Remove, "AzureRmAlertRule"), OutputType(typeof(List<AzureOperationResponse>))]
+    [Cmdlet(VerbsCommon.Remove, "AzureRmAlertRule"), OutputType(typeof(PSObject))]
```

Comment:
I'll return List<Microsoft.Azure.AzureOperationResponse> with a singe element.
The element contains: status = OK, and an empty request Id.
This is since the new rules for the specs are that remove operations return nothing. This is still sort of a breaking change I know.
Returning a list is a bug btw, but I'll keep it like that for backwards compatibility.